.github/workflows/docs.yaml

```diff
@@ -0,0 +1,21 @@
+name: Publish docs
+on: [push]
```

Comment:
Should it only run on push to the master?

Reply:
No, it publishes a preview on other branches